### ISSUE TITLE
SC 3.2.6: Is there a need to define "Programmatic notification"?

### ISSUE THREAD

**Issue by spanchang** [w3c/wcag21#526]
The term “programmatically set” is already used and defined in WCAG 2.0. This term should be used rather than defining a new one. This covers UA/At support.
The actual manner  in which notification is done depends on platform-At compatibility and nuances of the AT.

**Comment by awkawk** [comment]
@spanchang Which SC is this for? 3.2.6 doesn't seem to be the right one?

**Comment by spanchang** [comment]
Sorry: Appears this definition was only being discussed for SC 3.2.6 but has not made it into the 2.1 draft.
